RoosterPlanner.Data/Repositories/ShiftsRepository.cs

```diff
@@ -23,14 +23,41 @@ public ShiftRepository(RoosterPlannerContext dataContext) : base(dataContext)
         {
         }
 
-        public Task<List<Shift>> GetActiveShiftsForProjectAsync(Guid projectId)
+        public Task<List<Shift>> GetAll(Guid projectId)
         {
-            return EntitySet.Where(i => i.Date > DateTime.Now
-            && i.Task.DeletedDateTime == null
-            && i.Task.ProjectTasks.Any(x => x.ProjectId == projectId))
-            .OrderBy(i => i.Date).ThenBy(i => i.StartTime)
-                .Include(t => t.Task)
-                .Include("Task.Category").ToListAsync();
+            if (projectId == Guid.Empty)
+                return null;
```

Comment:
Omdat hier een Task wordt terug gegeven, beter om Task.CompletedTask of Task.FromResult<>(null of new List()) terug te geven